docs/macros.py

```diff
@@ -22,12 +23,14 @@ def define_env(env):
     This is the hook for defining variables, macros and filters.
     """
 
-    def indent(input_string: str):
-        return input_string.replace("\n", "\n\t")
+    def indent(input_string: Optional[str]) -> Optional[str]:
+        if input_string:
```

Comment:
Can be simplified to `return input_string and input_string.replace("\n", "\n\t")`

Reply:
you probably meant `or`, otherwise it returns None.
I will update `return input_string or input_string.replace("\n", "\n\t")`